packages/bsky/src/hydration/actor.ts

```diff
@@ -116,18 +115,19 @@ export class ActorHydrator {
       ) {
         return acc.set(did, null)
       }
-      const profile =
-        includeTakedowns || !actor.profile?.takenDown
-          ? actor.profile
-          : undefined
+
+      const profile = actor.profile
+        ? parseRecord<ProfileRecord>(actor.profile, includeTakedowns)
+        : undefined
+
       return acc.set(did, {
         did,
         handle: parseString(actor.handle),
-        profile: parseRecordBytes<ProfileRecord>(profile?.record),
+        profile: profile?.record,
         profileCid: profile?.cid,
-        profileTakedownRef: safeTakedownRef(profile),
-        sortedAt: profile?.sortedAt?.toDate(),
-        indexedAt: profile?.indexedAt?.toDate(),
+        profileTakedownRef: profile?.takedownRef,
```

Comment:
Any reason to drop usage of `safeTakedownRef()` here?  It has a fallback for `profile?.takedownRef`, so the logic is a little different.

Reply:
`parseRecord` does exactly that 👍 
similar to how we got to drop `toDate` on sortedAt/indexedAt